extensions/wikia/ContentReview/scripts/contentReviewDiffPage.js

```diff
@@ -40,6 +40,8 @@ define(
 							response.notification,
 							'confirm'
 						);
+						$('.content-review-diff-button').hide();
```

Comment:
Regarding the double request that @kamilkoterba has pointed out - I think that the buttons should be hidden *immediately* after one of them is clicked. Then, in case of an error, they could reappear. The similar behavior is used in Special:Flags to prevent such behavior.